receiver/m365receiver/README.md

```diff
@@ -1,8 +1,15 @@
+# Microsoft 365 Receiver
+Receives metrics from [Microsoft 365](https://www.microsoft365.com/)
 via the [Microsoft Graph API](https://learn.microsoft.com/en-us/graph/api/overview?view=graph-rest-1.0&preserve-view=true),
 and logs via the [Microsoft Management API](https://learn.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema).
 
+## Important Notes
+Due to the nature of the APIs being used to collect data from Microsoft, there are some important notes that should be acknowledged before continuing.
+
+- Metrics collected by this receiver can be 48 hours old. This is because the Reports endpoint of the Microsoft Graph API can take up to 48 hours to be updated with new data. The timestamps for metrics will represent when the data was scraped from the API rather than when the data points occurred (48 hours earlier). Because of this, the metrics receiver has a strict scrape interval of 1 hour that cannot be changed. This is done so that the receiver can scrape data from the API once it becomes available, within a reasonable amount of time. This also means that most data points are duplicates. Please keep this in mind when viewing metric data. For more info please see Microsoft's [documentation](https://learn.microsoft.com/en-us/microsoft-365/admin/activity-reports/activity-reports?view=o365-worldwide#how-to-get-to-the-reports-dashboard).
+- When an instance of Microsoft 365 is going to have logs collected for the very first time, it can take several hours for logs to be made available by the API. After completing the required step (described in further detail [here](#configuring-microsoft-365)), wait 60 minutes before attempting to collect logs. It takes 60 minutes for activation to be completed, during which Microsoft will make it seem like logs are still disabled. Once logs are completely activated, it can still take up to 12 hours before any logs are available for consumption. During this time, logs can be "collected" but the API won't have any logs available. After this initial waiting period logs will begin to appear "regularly". Microsoft gives no guarantee on when logs will be made available in the API after they've occurred, but it's been observed to take around 3 hours in the worst case. This may change though depending on the tenant being monitored. For more info please see Microsoft's [documentation](https://learn.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-reference#working-with-the-office-365-management-activity-api).
+- As part of [configuring Microsoft 365](#configuring-microsoft-365) a client secret will need to be generated in Azure Active Directory. When this is done, a date for when the secret expires will need to be selected. Remember and keep track of this date in some manner. When the client secret expires the receiver will fail and need to be restarted with a newly generated client secret. 
```

Comment:
> Remember and keep track of this date in some manner. When the client secret expires the receiver will fail and need to be restarted with a newly generated client secret.
I think it's clearer to just say that 'the user is responsible for tracking expiration and acquiring new secrets'